src/components/PostcodeLookup/PostcodeLookup.js

```diff
@@ -359,8 +359,9 @@ class PostcodeLookup extends Component {
       invalidErrorText: 'Please enter a valid postcode',
       emptyFieldErrorText: 'Please enter your postcode',
       extraClass: 'search-box',
+      autocomplete: 'off',
```

Comment:
I noticed that in chrome this is still allowing autofill by the way.
Seems to be a bug: https://caniuse.com/#feat=input-autocomplete-onoff

So if autofill in chrome is filling in the postcode field with an address that's less than 15 characters it will go through.
Not sure if this is then still worth putting in..

Reply:
@Heleen-cr are we not setting the max length on the field?

<input type="text" name="usrname" maxlength="10"><br>

Reply:
the min an max properties are there for number fields. 
We're not setting a max length as a property on our inputFields no. 
But we recently updated the valdiation for the Postcode field to not accept more than 15 characters, so we can reduce that or decide to put in a maxLength property (the latter being a bit more work)

Reply:
For it to work in chrome the value must be `true` or `false`